source/sessions/driver-sessions.rst

```diff
@@ -515,7 +519,7 @@ includes at least one connected, data-bearing server
 
 2. Having verified in step 1 that the ``TopologyDescription`` includes at least
 one connected server a driver can now determine whether sessions are supported
-by inspecting the ``logicalSessionTimeoutMinutes`` property
+by inspecting the ``TopologyType`` and ``logicalSessionTimeoutMinutes`` property
```

Comment:
When this was first written we were not expecting the isMaster response from a standalone to include a logicalSessionTimeoutMinutes.

Why does a standalone return that if it doesn't support sessions?

This seems like a good workaround though. Although if standalones start supporting sessions this will have to be revisited.

Reply:
Standalones don't include it.  I phrased it this way to highlight (again) that this is all irrelevant for standalones (allowing a driver to shortcut on topology type, too).

Reply:
Standalones *do* include logicalSessionTimeoutMinnutes in the isMaster respone:

```javascript
> db.isMaster()
{
        "ismaster" : true,
        "maxBsonObjectSize" : 16777216,
        "maxMessageSizeBytes" : 48000000,
        "maxWriteBatchSize" : 100000,
        "localTime" : ISODate("2018-01-12T15:07:53.611Z"),
        "logicalSessionTimeoutMinutes" : 30,
        "minWireVersion" : 0,
        "maxWireVersion" : 6,
        "readOnly" : false,
        "ok" : 1
}
```

Reply:
You're right!  I had spot checked it, but apparently my 3.6 was still in FCV 3.4 mode.  Blew away the database and restarted and now I see it.